featurebyte/schema/target_namespace.py

```diff
@@ -29,17 +29,25 @@ class TargetNamespaceCreate(FeatureByteBaseModel):
     default_target_id: PydanticObjectId
     default_version_mode: DefaultVersionMode = Field(default=DefaultVersionMode.AUTO)
     entity_ids: List[PydanticObjectId]
-    horizon: str
+    horizon: Optional[str]
 
 
 class TargetNamespaceUpdate(BaseDocumentServiceUpdateSchema):
     """
     TargetNamespace update schema
     """
 
-    target_ids: Optional[List[PydanticObjectId]]
     default_version_mode: Optional[DefaultVersionMode]
     default_target_id: Optional[PydanticObjectId]
+    horizon: Optional[str]
+
+
+class TargetNamespaceServiceUpdate(TargetNamespaceUpdate):
+    """
+    TargetNamespaceService update schema
+    """
+
+    target_ids: Optional[List[PydanticObjectId]]
```

Comment:
`TargetNamespaceUpdate` schema is used in API endpoint.
`TargetNamespaceServiceUpdate` is used in service level.

Reply:
probably worth documenting as a comment in code as well so that it's clearer to others reading the code